From c7877a9547c8b08200f32de233ac3ff5426178b5 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 24 Aug 2021 22:46:47 +0300
Subject: [PATCH] [SW-36150] gaudi2: pll: white space fixes

Remove empty line and align PLL macros values.
No funcitonal changes, only white space.

Change-Id: Ia73f75b7a0f4b9faab9b4e211585538f50904256
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/common/pll.c b/arch/gaudi2/common/pll.c
index 4940a61..172bec2 100644
--- a/arch/gaudi2/common/pll.c
+++ b/arch/gaudi2/common/pll.c
@@ -50,11 +50,10 @@
 	return out_freq;
 }
 
-#define GAUDI2_PLL_CFG		0x100
-#define GAUDI2_PLL_CFG_EN	BIT(0)
-#define GAUDI2_PLL_CFG_LOCK	BIT(16)
+#define GAUDI2_PLL_CFG			0x100
+#define GAUDI2_PLL_CFG_EN		BIT(0)
+#define GAUDI2_PLL_CFG_LOCK		BIT(16)
 #define GAUDI2_PLL_DIV_SEL_DELAY	0x1E0
-
 
 void hl_pll_reset(struct hl_pll_cfg *cfg, bool reset)
 {
